app/services/curation_concerns/workflow/confirm_submission.rb

```diff
@@ -0,0 +1,30 @@
+module CurationConcerns
+  module Workflow
+    class ConfirmSubmission
+      # params consists of
+      #   entity: entity
+      #   comment: comment
+      #   user: user
+      #   recipients: recipients(notification)
+      def self.send_notification(params = {})
+        comment = params[:comment].comment
+
+        notify_users = params[:recipients]["to"] + params[:recipients]["cc"]
+        notify_users = notify_users.uniq
+
+        updated_at = params[:entity].updated_at
+        user_email = params[:user].email
+
+        # title and id
+        id = params[:entity].proxy_for_global_id.sub(/.*\//, '')
+        g = GenericWork.find id
```

Comment:
Don't hard code GenericWork here. Not all works are GenericWork. Use `Sipity::Entity#proxy_for` instead.